web/src/components/AirgapUploadProgress.jsx

```diff
@@ -45,44 +46,47 @@ function AirgapUploadProgress(props) {
 
   let progressBar;
   let percentage;
+  let uploadComplete;
 
   if (total > 0 && sent > 0) {
+    uploadComplete = sent === total
     percentage = Math.floor((sent / total) * 100).toFixed() + "%";
     progressBar = (
-      <div className={`progressbar ${smallSize && "small"}`}>
-        <div className="progressbar-meter" style={{ width: `${(sent / total) * (smallSize ? 100 : 600)}px` }} />
+      <div className={`progressbar ${smallSize ? "small" : ""}`}>
+        <div className={`progressbar-meter ${uploadComplete ? "complete" : ""}`} style={{ width: `${(sent / total) * (smallSize ? 100 : 600)}px` }} />
       </div>
     );
   } else {
     percentage = "0%";
     progressBar = (
-      <div className={`progressbar ${smallSize && "small"}`}>
+      <div className={`progressbar ${smallSize ? "small" : ""}`}>
         <div className="progressbar-meter" style={{ width: "0px" }} />
       </div>
     );
   }
 
-  props.data?.startPolling(2000);
-
+  props.data?.startPolling(1000);
+  
   let statusMsg = getAirgapInstallStatus?.currentMessage;
   try {
     // Some of these messages will be JSON formatted progress reports.
     const jsonMessage = JSON.parse(statusMsg);
+    console.log(jsonMessage);
```

Comment:
clean console.log

Reply:
:/ I removed this and apparently didn't save it before commiting...

Reply:
:)